includes/classes/class-rtcamp-google-embeds.php

```diff
@@ -261,15 +270,17 @@ private function get_thumbnail_url( $file_id, $user_id = false ) {
 		// Check if a preview exists for supplied file id.
 		$thumbnail_url = sprintf( 'https://drive.google.com/thumbnail?id=%s&sz=w400-h400', $file_id );
 		$response      = wp_remote_get( $thumbnail_url );
+		$contents      = wp_remote_retrieve_body( $response );
```

Comment:
We should download file if `$status_code` is 200, before `$this->save_thumbnail`